### PR TITLE
Improves readme

### PR DESCRIPTION
**Pull request by rafael**
# Desc
* Improves readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,91 @@
...
+  *   NYC: 
+      *   POC: Ibrahim Rodriguez
+      *   Slack Channel: [#c4v-nyc](https://codeforvenezuela.slack.com/messages/CHJB5NJP8)
+  *   Barcelona ([RSVP](https://welcu.com/codeforvenezuelaeu/hackathon-code-for-venezuela)):
+      *   POC: Luis Castro
+      *   Slack Channel: [#c4v-bcn](https://codeforvenezuela.slack.com/messages/CHEFHUDCY)
+  *   Bogota: 
+      *   POC: Anibal Rojas
+      *   Slack Channel: [#c4v-bog](https://codeforvenezuela.slack.com/messages/CHEFD9KMW)
+
```
Comment (edgar) [inline README.md:84]: Do we have the physical address for the venues? should we add them?

**Review by edgar — COMMENTED** [review]
👍  just a NB comment about the venues

File: README.md [hunk 2]
```diff
@@ -2,12 +2,93 @@
...
+TODO 
+
+## During the Event
+
+1. **Teams**: 
+    1. Make sure to have a team. 
+    2. We want to bring together: developers, PMs, Data scientists, designers etc. 
+    3. A team could be made up of engineers only or a mix of cross-discipline teammates. 
+    4. We encourage freedom to innovate and be creative. 
+    5. Team Size Recommendation: 3 to 5 people.
```
Comment (gustavoatt) [inline README.md]: I would reword this as: `Recommended team size: ...`

File: README.md [hunk 3]
```diff
@@ -2,12 +2,93 @@
...
+
+If you want to work on a challenge that is not by our partner organization, you are welcome to do so! Share you challenge in [#hackathon-ga](https://codeforvenezuela.slack.com/messages/CH5JMFBB2).
+
+## Demos and Presentations
+
+1. Each team will present a 3 minute demo on their work. 
+2. You can record a video and we will present it on site as a part of the final presentations. 
+3. **Deadline**: Sunday 14, 4:30pm Pacific Time) 
+4. **Software**:
+    1. Our suggestion is to use Loom to record the video of the demo: [https://www.useloom.com](https://www.useloom.com)
```
Comment (gustavoatt) [inline README.md]: nit: Perhaps reword to: `We suggest using Loom to record the video of the demo...`

File: README.md [hunk 4]
```diff
@@ -2,12 +2,93 @@
...
+5. Post your video via Slack in the channel [#hackathon-ga](https://codeforvenezuela.slack.com/messages/CH5JMFBB2)
```
Comment (gustavoatt) [inline README.md]: `Post your video in Slack [#hackathon-ga](...)`.

File: README.md [hunk 5]
```diff
@@ -2,12 +2,93 @@
...
+
+## Demos and Presentations
+
+1. Each team will present a 3 minute demo on their work. 
+2. You can record a video and we will present it on site as a part of the final presentations. 
+3. **Deadline**: Sunday 14, 4:30pm Pacific Time) 
+4. **Software**:
+    1. Our suggestion is to use Loom to record the video of the demo: [https://www.useloom.com](https://www.useloom.com)
+5. Post your video via Slack in the channel [#hackathon-ga](https://codeforvenezuela.slack.com/messages/CH5JMFBB2) 
+6. If possible, you are invited to watch live the feedback from our panelist. This will happen around 4:30PM PST,  Sunday 14.
```
Comment (gustavoatt) [inline README.md]: I would remove the `If possible` here. We are inviting folks to watch the live feedback, they can decide whether they would take that invitation.

**Review by gustavoatt — DISMISSED** [review]
Looks good to me modulo a few minor comments.

Thank you for taking the time to write this Rafael! 🚢

**Review by gustavoatt — APPROVED** [review]
🚢
